internal/registry/credentialrequest/rest_test.go

```diff
@@ -27,14 +27,27 @@ import (
 	"go.pinniped.dev/internal/testutil"
 )
 
+func TestRestMethods(t *testing.T) {
+	t.Parallel()
+	type storage interface {
+		rest.Creater
+		rest.NamespaceScopedStrategy
+		rest.Scoper
+		rest.Storage
+	}
+	require.Implements(t, (*storage)(nil), new(REST))
+}
+
 func TestNew(t *testing.T) {
+	t.Parallel()
 	r := NewREST(nil, nil)
 	require.NotNil(t, r)
 	require.True(t, r.NamespaceScoped())
 	require.IsType(t, &loginapi.TokenCredentialRequest{}, r.New())
 }
 
 func TestCreate(t *testing.T) {
+	t.Parallel()
```

Comment:
uh this is spec test so I think it needs a fix below

Reply:
Whoops, I ran it once with `-race` and it worked, but I don't want to cause issues. I'll just revert that bit.